### DISCUSSION TITLE
Is it possible to "mark" a spec directory as "must not fail"?

### DISCUSSION THREAD

**Discussion by BloodyRain2k** [cypress-io/cypress#28110] [Questions and Help]
I'm using Cypress to test a SPFX app and I've needed to write a bunch of custom routines for that.

This resulted in a spec structure where I have 3 directories in "Cypress/e2e": "/core", "/normal", and "/exchange".

The "/core" directory is basically just unit tests for my custom routines to ensure that changes I made somewhere didn't break anything.
This means that ANY failure in these specs technically invalidates all other tests, even though in most cases it just breaks them.

And when I'm running a full test, I'm doing it through the command line with
`npx cypress run -r spec -s .\cypress\e2e\core,.\cypress\e2e\normal\,.\cypress\e2e\exchange\`

Is there any way of telling Cypress that it doesn't have to bother with the following specs, if anything in "/core" failed?

**Comment by hansmorb** [comment]
I think what you are looking for is failfast and there is a Cypress plugin that does just that:

https://github.com/javierbrea/cypress-fail-fast
